jobs/rabbitmq-server/templates/config.erb

```diff
@@ -5,5 +5,5 @@ RMQ_FD_LIMIT="${RMQ_FD_LIMIT:?must be set}"
 SKIP_PREPARE_FOR_UPGRADE=true
 <% end %>
 <% if p('rabbitmq-server.ssl.inter_node_enabled') -%>
-RABBITMQ_CTL_ERL_ARGS="-proto_dist inet_tls -ssl_dist_optfile /var/vcap/jobs/rabbitmq-server/etc/inter_node_tls.config"
+CTL_ERL_ARGS="-pa $ERL_SSL_PATH -proto_dist inet_tls -ssl_dist_optfile /var/vcap/jobs/rabbitmq-server/etc/inter_node_tls.config"
```

Comment:
@MirahImage @Zerpet -

I'm not sure that this is necessary, given that `CTL_ERL_ARGS` is set in `rabbitmq-env.conf`.

What is the purpose of this `config` file?

Reply:
It appears to be only used by the prepare-for-upgrade script. I suspect we could remove this file and the change the upgrade-preparation script to template only if that is required (which is basically never, at this point, I believe it was used for the 3.6 to 3.7 upgrade).

Reply:
Agreed, we should template in `rabbitmq-server.init` directly, since nothing else is sourcing the `config` file. We may need to keep the `CTL_ERL_ARGS` because `rabbitmq-server.init` runs `rabbitmqctl` commands; or we could make sure that this script has the correct `RABBITMQ_HOME`, so that it loads `rabbitmq-env.conf` correctly.

Reply:
Yep, all of the `rabbitmqctl` invocations in `rabbitmq-server.init` failed in the latest env that @Zerpet provided for me to use, that uses inter-node TLS. I'll look into fixing this.